flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNodeConfig.java

```diff
@@ -97,18 +87,18 @@ public long getStateRetentionTime() {
 
     // See https://issues.apache.org/jira/browse/FLINK-26190
     /**
-     * Using {@link #originalTableConfig} to satisify tests like {@code OverAggregateHarnessTest},
-     * which use {@code HarnessTestBase#TestTableConfig} to individually manipulate the
+     * Using {@link #plannerConfig} to satisify tests like {@code OverAggregateHarnessTest}, which
+     * use {@code HarnessTestBase#TestTableConfig} to individually manipulate the
      * maxIdleStateRetentionTime. See {@link TableConfig#getMaxIdleStateRetentionTime()}.
      */
     @Deprecated
     public long getMaxIdleStateRetentionTime() {
-        return originalTableConfig.getMaxIdleStateRetentionTime();
+        return plannerConfig.getMaxIdleStateRetentionTime();
     }
 
     // See https://issues.apache.org/jira/browse/FLINK-26190
     /** See {@link TableConfig#getLocalTimeZone()}. */
     public ZoneId getLocalTimeZone() {
-        return tableConfig.getLocalTimeZone();
+        return plannerConfig.getLocalTimeZone();
```

Comment:
can't we use the `TableConfigUtils` now and include the `nodeConfig` with this as well?